content/intro/getting-started/index.md

```diff
@@ -36,13 +34,47 @@ Once you have submitted your email, you should receive a confirmation page that
 
 ![Sign Up Page 3](signup3.png)
 
+### Check your email
+
 Next, check your email for a link to confirm your account and create a password.
+
 In your email, you will find the following link. Click the **Set Password** button to set the password of your BluBracket login account.
 
-**Note**: *This login link will be how you will access your BluBracket account.
+NOTE: This password request expires in 48 hours.
 
 ![Sign Up Page 4](signup4.png)
 
 Once you have set your password, you are ready to get started with connecting your repositories to BluBracket.
 
 ![Sign Up Page 5](signup5.png)
+
+### Set your password
+
+[set password screenshot]
+
+Fill in your email address and password.
+
+**Password Criteria**:
+
+- Passwords must be at least eight characters.
+- Passwords must include at least 1 number, a special character or symbol, and one uppercase letter.
+
+## Signing in
+
+Refer to your welcome email when you first signed in to BluBracket for your personalized dashboard URL. You must use this URL in order to sign in.
+
+[email screenshot]
+
+[login page screenshot]
+
+## Adding your first code server
+
+Go to Settings > Code Servers in the BluBracket web console to add one or more code repositories hosted on GitHub, GitLab, Bitbucket, or another supported code server.
+
+Full details about [how to add a code server are available in the usage guide](https://docs.blubracket.com/how-to/add-code-servers/).
+
+## Improve your code security with every commit
+
+1. Add BluBracket to your workflow with the CLI tool, GitHub Checks (or other CI checks) so you can identify risks before your next commit.
+2. Review existing risks to understand your overall code health and make a plan for what is urgent and what can wait.
```

Comment:
Consider this link here? https://docs.blubracket.com/intro/key-workflows/#understanding-and-taking-action-on-existing-risks